tendrl/commons/objects/cpu/__init__.py

```diff
@@ -1,3 +1,4 @@
+import psutil
 from tendrl.commons import objects
```

Comment:
you need to separate imports by 2 blank lines
- python stdlibs
- third party (like psutil)
- imports from tendrl

Reply:
ack